app/src/main/java/org/oppia/app/topic/info/TopicInfoFragmentPresenter.kt

```diff
@@ -63,7 +63,6 @@ class TopicInfoFragmentPresenter @Inject constructor(
       /* attachToRoot= */ false
```

Comment:
Not able to comment on that particular line but writing here. 
There is an unused import at line number 13 `import org.oppia.app.model.EventLog`

Reply:
Done.